src/main/java/com/onarandombox/MultiverseCore/configuration/MVConfigNodes.java

```diff
@@ -0,0 +1,170 @@
+            .build());
+
+    public static final MVValueNode<Integer> CUSTOM_PORTAL_SEARCH_RADIUS = node(MVValueNode.builder("portal.custom-portal-search-radius", Integer.class)
+            .comment("")
+            .comment("This config option defines the search radius Multiverse should use when searching for a portal.")
+            .comment("This only applies if use-custom-portal-search is set to true.")
+            .defaultValue(128)
+            .build());
+
+    private static final MVCommentedNode MESSAGING_HEADER = node(MVCommentedNode.builder("messaging")
+            .comment("")
+            .comment("")
+            .build());
+
+    public static final MVValueNode<Boolean> ENABLE_CHAT_PREFIX = node(MVValueNode.builder("messaging.enable-chat-prefix", Boolean.class)
+            .comment("This config option defines whether or not Multiverse should prefix the chat with the world name.")
+            .comment("This only applies if use-custom-portal-search is set to true.")
+            .defaultValue(false)
+            .build());
+
+    public static final MVValueNode<String> CHAT_PREFIX_FORMAT = node(MVValueNode.builder("messaging.chat-prefix-format", String.class)
+            .comment("")
+            .comment("This config option defines the format Multiverse should use when prefixing the chat with the world name.")
+            .comment("This only applies if enable-chat-prefix is set to true.")
+            .defaultValue("[%world%]%chat%")
+            .build());
+
+    private static final MVCommentedNode MISC_HEADER = node(MVCommentedNode.builder("misc")
+            .comment("")
+            .comment("")
+            .build());
+
+    public static final MVValueNode<Integer> GLOBAL_DEBUG = node(MVValueNode.builder("misc.global-debug", Integer.class)
+            .comment("This is our debug flag to help identify issues with Multiverse.")
+            .comment("If you are having issues with Multiverse, please set this to 3 and then post your log to pastebin.com")
+            .comment("Otherwise, there's no need to touch this. If not instructed by a wiki page or developer.")
+            .comment("  0 = Off, No debug messages")
+            .comment("  1 = fine")
+            .comment("  2 = finer")
+            .comment("  3 = finest")
+            .defaultValue(0)
+            .build());
+
+    public static final MVValueNode<Boolean> SILENT_START = node(MVValueNode.builder("misc.silent-start", Boolean.class)
+            .comment("")
+            .comment("If true, the startup console messages will no longer show.")
+            .defaultValue(false)
+            .build());
+
+    public static final MVValueNode<Boolean> I_DONT_WANT_TO_DONATE = node(MVValueNode.builder("misc.i-dont-want-to-donate", Boolean.class)
+            .comment("")
+            .comment("If you don't want to donate, you can set this to true and Multiverse will stop nagging you.")
+            .defaultValue(false)
+            .build());
```

Comment:
What if we moved the donation message to the top of the config file instead of logging in onEnable?

Reply:
Uh that's your call, the donation are to you haha. I'm ok with changing it.